packages/playground-examples/copy/ko/JavaScript/External APIs/TypeScript with Node.js

```diff
@@ -0,0 +1,52 @@
+//// { order: 3, isJavaScript: true }
+
+// Node.js는 Chrome 브라우저를 지원하는 JavaScript 엔진인 v8을 기반으로
+// 구축된 매우 유명한 JavaScript 런타임 기술입니다. 이것을 사용하면
```

Comment:
```suggestion
// 구축된 매우 유명한 JavaScript 런타임입니다. 이것을 사용하면
```
[node.js](https://nodejs.org/ko/) 번역을 따라가도 될 것 같아요 :)